pulpcore/cli/common/generic.py

```diff
@@ -145,6 +145,19 @@ def _name_callback(
     return value
 
 
+def _id_callback(
+    ctx: click.Context, param: click.Parameter, value: Optional[str]
+) -> Optional[str]:
+    if value is not None:
+        entity_ctx = ctx.find_object(PulpEntityContext)
+        assert entity_ctx is not None
+        if value.startswith("/"):
+            entity_ctx.pulp_href = value
```

Comment:
Originally I started by implementing an `id_option` that could take an `href_pattern` but that maybe seemed like overkill. I'm tempted to do it but I thought I'd start with something simple first.